ts/features/zendesk/store/reducers/index.ts

```diff
@@ -47,11 +54,13 @@ export type ZendeskState = {
   ticketNumber: pot.Pot<number, Error>;
   getSessionPollingRunning?: boolean;
   getZendeskTokenStatus?: ZendeskTokenStatusEnum | "401";
+  zendeskMap: ZendeskSubcategoriesErrorsConfig;
```

Comment:
What do you think about renaming `zendeskMap` to something more specific? For example, would a name like `zendeskSubcategoriesErrorMap` be fine?

Reply:
Renamed here:  https://github.com/pagopa/io-app/pull/6522/commits/0959a516e2d50e73642a903a368ef28a66a238e3